File: Q17/main.c
#include<stdio.h>
int main(){
    int row1,col1;
    scanf("%d%d",&row1,&col1);
    int arr1[row1][col1];
    for(int i=0;i<row1;i++){
        for(int j=0;j<col1;j++){
            scanf("%d",&arr1[i][j]);
        }
    }
    
    int row2,col2;
    scanf("%d%d",&row2,&col2);
    int arr2[row2][col2];
    for(int i=0;i<row2;i++){
        for(int j=0;j<col2;j++){
            scanf("%d",&arr2[i][j]);
        }
    }
    int res[row1][col2];

    for(int i=0;i<row1;i++){
        for(int j=0;j<col2;j++){
            res[i][j]=0;
            for(int k=0;k<col1;k++){
                res[i][j]+=(arr1[i][k]*arr2[k][j]);
            }
        }
    }

    for(int i=0;i<row1;i++){
        for(int j=0;j<col2;j++){
            printf("%d ",res[i][j]);
        }
        printf("\n");
    }
    return 0;
}

File: Q17/main.py
row1,col1=map(int,input().split())
arr1=[]
for i in range(row1):
    arr1.append(list(map(int,input().split())))

row2,col2=map(int,input().split())
arr2=[]
for i in range(row2):
    arr2.append(list(map(int,input().split())))

res=[[0]*col2 for i in range(row1)]

for i in range(row1):
    for j in range(col2):
        res[i][j]=0
        for k in range(col1):
            res[i][j]+=(arr1[i][k]*arr2[k][j])

for i in range(row1):
    for j in range(col2):
        print(res[i][j],end=" ")
    print()